packages/kolibri-common/components/SearchBox.vue

```diff
@@ -117,15 +121,29 @@
           textAlign: this.isRtl ? 'right' : '',
         };
       },
+      computedPlaceholder() {
+      if (this.viewportWidth < 1264 && this.viewportWidth >= 840) {
```

Comment:
Here instead of checking with these specific pixels, lets instead do this check using our [useKResponsiveWindow composable](https://design-system.learningequality.org/usekresponsivewindow). We can use the `windowBreakpoint` property, and lets show the "find" string if this windowBreakpoint is 3 or 4 which are the corresponding breakpoints.